script/Arcueid/Day 2/02_00_ARC02_6.txt

```diff
@@ -91,7 +91,7 @@ I can still hear the dog howling outside my window.
 [sha:322aa62f8b75fbcabecefd78ce242a2653ddca08]{
 -- Page 342, Offset 3508.
 -- 時計は夜の11時になったばかり。
-The clock has just turned eleven.
+The clock has just turned 11.
```

Comment:
I'd say keep eleven here, it's a 'low' number. Also maybe 
> The clock has just struck eleven.

because a clock itself doesn't really turn,

Reply:
so what's the limit we're following then? below a hundred keep it in words?

Reply:
It's hard to pin down language with hard and fast rules. I'd say generally under a hundred sure, but it also changes based on the context of the words - e.g. if it's a 'precise' or 'scientific' context, I'd be more likely to use numbers.

Reply:
alright i'll keep that in mind then